FhirToDataLake/docs/FHIR-Data-Mapping.md

```diff
@@ -78,41 +78,46 @@ Example:
 "extension": "[{\"url\": \"http://nema.org/fhir/extensions#0010:1010\",\"valueQuantity\": {\"value\": 56,\"unit\": \"Y\"}}]"
 }
 ```
-   
 
-### Exclude primitive extensions and inline resources
+**Inline Resources**
 
-Extensions on primitive types usually begin with a underline prefix in the raw FHIR JSON data. Inline [Resources](https://www.hl7.org/fhir/resource.html#Resource) such as "contained" and "outcome" properties are flexiable FHIR resource data properties. These two kinds of fields are excluded in Parquet files.
+Inline [Resources](https://www.hl7.org/fhir/resource.html#Resource) such as "contained" and "outcome" properties are flexible FHIR resource data properties, and will be wrapped into a single string.
+
+Example:
 
 ```javascript
 /* Raw FHIR Json data */
 {
-"resourceType": "Patient",
-"id": "example",
-"gender": "male",
-// Primitive extensions will excluded
-"_gender": {
-    "extension": [{
-        "url": "http://nema.org/examples/extensions#gender",
-        "valueCoding": {
-            "code": "M"
-    }}]
-},
-// Inline resources will be excluded
-"contained" : {...}
+  "resourceType" : "Condition",
+  "contained": [
+    {
+      "resourceType" : "Practitioner",
+      "id" : "p1",
+      "name" : [{
+        "family" : "Person",
+        "given" : ["Patricia"]
+      }]
+    }],
+   "asserter" : {
+     "reference" : "#p1"
+  }
 }
 
 /* Converted data in parquet file */
 {
-"resourceType": "Patient",
-"id": "example",
-"gender": "male",
+    "resourceType": "Condition",
+    "id": "example",
+    // Contained properties will be wrapped into single Json string
+    "contained": "[{\"resourceType\":\"Practitioner\",\"id\":\"p1\",\"name\":[{\"family\":\"Person\",\"given\":[\"Patricia\"]}]}]",
+    "asserter": {
+        "reference" : "#p1"
+    }
 }
 ```
```

Comment:
Shall we mention that if customer can use custom schema to get properties from contained resource like extensions? @RuiyiC